press/api/client.py

```diff
@@ -289,15 +299,6 @@ def check_team_access(doctype: str, name: str):
 	raise_not_permitted()
 
 
-def check_dashboard_actions(doctype, method):
```

Comment:
should have put the `is method dashboard whitelisted` logic in this function instead of duplicating the logic at multiple places